etl/steps/data/garden/democracy/2024-03-07/bmr.meta.yml

```diff
@@ -0,0 +1,293 @@
+        title: |-
+          <% if category == '91+' %>
+          Number of democracies (incl. women's suffrage) aged 91 or older
+          <% else %>
+          Number of democracies (incl. women's suffrage) aged << category >> years
+          <% endif %>
+        unit: "countries"
+        description_short: |-
+          <% if category == '91+' %>
+          The number of countries that have been democracies (including women's suffrage) for 91 or more years straight.
+          <% else %>
+          The number of countries that have been democracies (including women's suffrage) for << category >> years straight.
+          <% endif %>
+        description_key:
+          - *dk_democracy_ws_1
+
+  population_regime:
+    common:
+      presentation:
+          grapher_config:
+            hasMapTab: false
+            selectedEntityNames:
+              - Africa
+              - Asia
+              - Europe
+              - North America
+              - Oceania
+              - South America
+    variables:
+      population_regime:
+        title: |-
+          <% if category == 0 %>
+          People living in non-democracies
+          <% elif category == 1 %>
+          People living in democracies
+          <% else %>
+          People living in countries without regime data
+          <% endif %>
+        unit: "people"
+        description_short: |-
+          <% if category == 0 %>
+          The number of people that live in countries that are not democracies.
+          <% elif category == 1 %>
+          The number of people that live in countries that are democracies.
+          <% else %>
+          The number of people that live in countries for which there is no data.
+          <% endif %>
+        description_key:
+          - *dk_democracy_1
+      population_regime_ws:
+        title: |-
+          <% if category == 0 %>
+          People living in non-democracies, including women's suffrage
+          <% elif category == 1 %>
+          People living in democracies, including women's suffrage
+          <% endif %>
+        unit: "people"
+        description_short: |-
+          <% if category == 0 %>
+          The number of people that live in countries that are not democracies with women's suffrage.
+          <% elif category == 1 %>
+          The number of people that live in countries that are democracies with women's suffrage.
+          <% endif %>
+        description_key:
+          - *dk_democracy_ws_1
+
+  population_regime_years:
+    common:
+      presentation:
+          grapher_config:
+            hasMapTab: false
+            selectedEntityNames:
+              - Africa
+              - Asia
+              - Europe
+              - North America
+              - Oceania
+              - South America
+    variables:
+      population_years_in_democracy_consec:
+        title: |-
+          <% if category == '91+' %>
+          People living in democracies aged 91 or older
```

Comment:
It could be a bit clearer that it is the democracy that is 91> years and not the people. I think how you have it in the description short makes it clear.